doc/source/WorkingWithScenes/LightLinking/index.md

```diff
@@ -0,0 +1,169 @@
+
+![The Linked Lights plug of an object](images/interfaceLinkedLightsPlug.png "The Linked Lights plug of an object")
+
+Once the _Linked Lights_ plug is toggled on, the `linkedLights` attribute is added to the object's location. The value of the _Linked Lights_ plug is a [set expression](../../Reference/ScriptingReference/SetExpressions/index.md), which can consist of individual light locations, set names, or both.
+
+> Important :
+> The _Linked Lights_ plug is an exclusive list. If you toggle the plug and leave the list blank, the object will not be illuminated by any lights **at all**. 
+> In order to link a particular light **in addition to** all default lights to an object, you must include `defaultLights` in the set expression, as in `defaultLights /group/light`.
+
+> Tip :
+> If your object is a member of a set, the filter on the StandardAttributes node could provide the set (via a SetFilter node) instead of the path location (via the PathFilter, shown above). Put another way, you can perform **set**-centric light linking. The end result would still be the same, in that specific objects are linked to specific lights.
+
+
+### By location ###
+
+To link a light by location, simply add the light's full location to the set expression. No additional node is needed.
+
+![A light linking set expression with a location](images/taskLightLinkingSetExpressionLocation.png "A light linking set expression with a location")
+
+> Important :
+> When linking a light by its location, the path you provide **must** be the final location of the light within the hierarchy **as delivered to the render node**. For example, if you link to a light at `/group/light`, but later in the graph you shuffle the hierarchy and the light exists at `/building/lights/light`, the renderer will look for the light at the original location, find nothing there, and then the link will fail. Therefore, you would unintuitively need to target a downstream location in an upstream part of the graph.
+
+
+### By set ###
+
+In setups with more than a few lights, managing a set expression with multiple light locations is prone to user error, and tedious to maintain. That's why it's usually better to link objects to one or more sets containing lights, as the term _set expression_ suggests. A light can be added to any number of custom sets using the _Sets_ plug on the light's node, or a separate Set node. Custom sets are separate and do not affect the "defaultLights" set.
+
+![A Set node downstream of a light node in the Graph Editor](images/interfaceLightSetGraphEditor.png "A Set node downstream of a light node in the Graph Editor")
+
+![A Set node in the Node Editor](images/interfaceLightSetNodeEditor.png "A Set node in the Node Editor")
+
+To link one or more lights by a set, simply add it to the set expression.
+
+![A light linking set expression with a set](images/taskLightLinkingSetExpressionSet.png "The light linking set expression with a set")
+
+
+## Example Scenario ##
```

Comment:
I would have preferred to put this section right after the intro, but it's quite long, and I didn't want to prevent users from getting to the most important bits of what and how as soon as possible. Does it fit here, or should it be moved up?